x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -1,14 +1,312 @@
 {
   "template": {
     "mappings": {
+      "dynamic_templates": [
+        {
+          "match_ip": {
+            "match_mapping_type": "string",
+            "match": "ip",
+            "runtime": {
+              "type": "ip"
+            }
+          }
+        },
+        {
+          "message": {
+            "path_match": "message",
+            "mapping": {
+              "type": "match_only_text"
+            }
+          }
+        },
+        {
+          "log_level": {
+            "path_match": "log.level",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "log_logger": {
+            "path_match": "log.logger",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "trace_id": {
+            "path_match": "trace.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "span_id": {
+            "path_match": "span.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "transaction_id": {
+            "path_match": "transaction.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "service_name": {
+            "path_match": "service.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "service_version": {
+            "path_match": "service.version",
```

Comment:
Runtime field by default? Could we use https://www.elastic.co/guide/en/elasticsearch/reference/current/version.html ?

Reply:
Good idea about using the version field! But I think it should be indexed by default. It can be really useful in automated correlations to see if a new version is more prone to logging errors, for example.